dbschema/progress-report.variance-explanation.esdl

```diff
@@ -0,0 +1,12 @@
+module ProgressReport {
+  type VarianceExplanation extending ProgressReport::Child, Engagement::Child {
```

Comment:
```suggestion
  type VarianceExplanation extending ProgressReport::Child {
```

I'd be ok moving this to the progress report file too as @bryanjnelson pointed out.